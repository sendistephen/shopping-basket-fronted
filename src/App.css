body {
  font-family: 'Montserrat', sans-serif;
}

.main {
  position: relative;
}
.main-left-side {
  position: absolute;
  width: 700px;
  height: 100vh;
  background-color: #dde6f0;
}
.main-right-side {
  position: absolute;
  top: 130px;
  left: 600px;
}
.main-login-btn {
  position: absolute;
  top: 20px;
  left: 85%;
  border-radius: 30px;
  border-color: #dde6f0;
  color: #5e7493;
  padding: 10px 46px;
}

input[type='text'],
input[type='email'],
input[type='password'] {
  color: #8795a1;
  background-color: #edf2f7;
  border: none;
  padding: 25px 20px;
  font-weight: 400;
}
.form-rounded {
  border-radius: 50px;
}
.btn-register {
  background-color: #2c2c33;
  border-radius: 50px;
  padding: 14px 20px;
  color: #fff;
}
.btn-register:hover {
  color: #edf2f7;
  background-color: #27272e;
}
.main-login-btn:hover {
  background-color: #dde6f0;
  border-color: #dde6f0;
  color: #2c2c33;
  font-weight: 400 !important;
}
