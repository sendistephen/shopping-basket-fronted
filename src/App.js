import React, { Component, Fragment } from 'react';
import './App.css';
import Register from './components/auth/Register';
import Login from './components/auth/Login';
import { ToastContainer } from 'react-toastify';
import 'react-toastify/dist/ReactToastify.css';
import { Switch, Route } from 'react-router-dom';

class App extends Component {
  render() {
    return (
      <Fragment>
        <ToastContainer />
        <Switch>
          <Route path='/login' component={Login} />
          <Route path='/' component={Register} />
        </Switch>
      </Fragment>
    );
  }
}

export default App;
