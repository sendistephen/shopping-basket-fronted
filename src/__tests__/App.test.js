import React from 'react';
import App from '../App';
import { shallow, mount } from 'enzyme';
import toJson, { shallowToJson } from 'enzyme-to-json';

describe('App component', () => {
  const wrapper = shallow(<App />);

  it('renders without crashing', () => {
    mount(<App />);
  });

  it('renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('renders the switch', () => {
    expect(wrapper.find('Switch')).toHaveLength(1);
  });
});
