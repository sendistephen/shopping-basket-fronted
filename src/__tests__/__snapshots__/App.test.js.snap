// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders properly 1`] = `
<Fragment>
  <ToastContainer
    autoClose={5000}
    bodyClassName={null}
    className={null}
    closeButton={
      <CloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="top-right"
    progressClassName={null}
    progressStyle={null}
    role="alert"
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/baskets"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/login"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Redirect
        exact={true}
        from="/login"
        to="/baskets"
      />
    </Switch>
  </BrowserRouter>
</Fragment>
`;
